src/test/resources/org/prebid/server/it/openrtb2/medianet/test-auction-medianet-request.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "test-request-id",
+  "imp": [
+    {
+      "id": "test-imp-id",
+      "banner": {
+        "w": 300,
+        "h": 250
+      },
+      "ext": {
+        "medianet": {
+          "refresh_count": 1,
```

Comment:
`refresh_count` is it just random field? Or it should be described in medianet.json?

Reply:
It was a random field. We have updated the required fields in the medianet.json.